### PR TITLE
Fix everything

### PR DESCRIPTION
**Pull request by BenWu**

### CODE REVIEW

**Review by imjching — APPROVED** [review]
:lgtm: :)

File: index.html [hunk 1]
```diff
@@ -4,27 +4,35 @@
...
+        justify-items: end;
+      }
     </style>
   </head>
   <body>
-    <p>engineer on the <a href="https://wiki.mozilla.org/Project_Fission">fission</a> team at mozilla</p>
-    <br />
-    <p>github: <a href="https://github.com/kashav">kashav</a>, <a href="https://github.com/kshvmdn">kshvmdn</a></p>
-    <p>contact: <a href="mailto:kshvmdn@gmail.com">kshvmdn@gmail.com</a><p>
+    <div class="main-container">
```
Comment (kashav) [inline index.html:31]: I'm trying to keep the environmental footprint of this project as low as possible, so I'd really prefer if we didn't introduce a separate class for this. It's the only `<div>` in the file, we can just select it directly with `div`.

File: index.html [hunk 2]
```diff
@@ -4,27 +4,35 @@
...
           word-wrap: break-word; }
       a { color: rgb(5, 5, 5); }
       a:hover { text-decoration: none;
                 background-color: lavenderblush; }
+      .main-container {
+        width: 100%;
+        text-align: end;
+        display: grid;
+        justify-items: end;
+      }
```
Comment (kashav) [inline index.html:27]: I **hate** giving css curlies their own line. Please change this.

**Review by kashav — CHANGES_REQUESTED** [review]

**Comment by BenWu** [comment]
ℐ 𝒹𝒾𝓈𝒶𝓰𝓇ℯℯ 𝚭𝚯𝚯𝚸

**Pull request closed without merging**